### PR TITLE
Refactor templates

### PR DESCRIPTION
**Pull request by cdrage**
**What kind of PR is this?**

/kind cleanup

**What does does this PR do / why we need it**:

Cleans up the tmeplates we currently have.

Few things:
 - "enhancement" kind does not exist, replaced by "code refactoring"
 - minor cleaup of spacing

**Which issue(s) this PR fixes**:

None, no issue was opened.

**How to test changes / Special notes to the reviewer**:

Look at the markdown changes :)

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Enhancement.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 name: Enhancement
 about: Suggest an enhancement for existing odo functionality
 ---
-[kind/Enhancement]
+
+/kind code refactoring
```
Comment (kadel) [inline .github/ISSUE_TEMPLATE/Enhancement.md]: I don't think that "code refactoring" and Enhancement are the same

Comment (cdrage) [reply .github/ISSUE_TEMPLATE/Enhancement.md]: Yes, I know, but there is no such thing as the "enhancement" label :(. What alternative do you suggest?

Comment (kadel) [reply .github/ISSUE_TEMPLATE/Enhancement.md]: We can either create it. Or I would even consider getting rid fo enhancement altogether.
Is't enchantment kind of small new feature?

Comment (cdrage) [reply .github/ISSUE_TEMPLATE/Enhancement.md]: Yeah.. that's what I thought too haha. I'm going to remove it.

**Comment by cdrage** [comment]
@kadel Updated!

**Comment by cdrage** [comment]
Ready for review @kadel @amitkrout @girishramnani

**Pull request merged**